src/pint/models/astrometry.py

```diff
@@ -80,15 +90,14 @@ def solar_system_geometric_delay(self, toas, acc_delay=None):
         available as 3-vector toa.xyz, in units of light-seconds.
         """
         tbl = toas.table
-        L_hat = self.ssb_to_psb_xyz_ICRS(epoch=tbl["tdbld"].astype(numpy.float64))
-        re_dot_L = numpy.sum(tbl["ssb_obs_pos"] * L_hat, axis=1)
+        L_hat = self.ssb_to_psb_xyz_ICRS(epoch=tbl["tdbld"].astype(np.float64))
+        re_dot_L = np.sum(tbl["ssb_obs_pos"] * L_hat, axis=1)
         delay = -re_dot_L.to(ls).value
-        if self.PX.value != 0.0 and numpy.count_nonzero(tbl["ssb_obs_pos"]) > 0:
+        if self.PX.value != 0.0 and np.count_nonzero(tbl["ssb_obs_pos"]) > 0:
             L = (1.0 / self.PX.value) * u.kpc
-            # TODO: numpy.sum currently loses units in some cases...
+            # TODO: np.sum currently loses units in some cases...
```

Comment:
Is this still true? You can't safely use units with np.sum?

Reply:
I'm pretty sure astropy fixed this with 4.0. (line is only here because I cleaned up numpy -> np)

Reply:
Also I rewrite this whole function in the actual solar wind PR #894